metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/metron_security.py

```diff
@@ -46,7 +47,7 @@ def storm_security_setup(params):
                   )
 
         File(ambari_format('{client_jaas_path}'),
-             content=Template('client_jaas.conf.j2'),
+             content=InlineTemplate(params.metron_client_jaas_conf_template),
```

Comment:
Instead of populating the template from the static .j2 file, the template now comes from the configuration. In MPack parlance, its now using an `InlineTemplate`.